src/cascadia/TerminalApp/JsonUtils.cpp

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "pch.h"
+#include "Utils.h"
+#include "JsonUtils.h"
+#include "../../types/inc/Utils.hpp"
+
+void TerminalApp::JsonUtils::GetOptionalColor(const Json::Value& json,
+                                              std::string_view key,
+                                              std::optional<uint32_t>& target)
+{
+    auto conversionFn = [](const Json::Value& value) -> uint32_t {

```

Comment:
I think `conversionFn` can be `const` the way down here as you're not changing its val.